services/static-webserver/client/source/class/osparc/store/StaticInfo.js

```diff
@@ -81,6 +81,18 @@ qx.Class.define("osparc.store.StaticInfo", {
         return wsStaticData[key];
       }
       return 30;
+    },
+
+    __getMetadata: function() {
+      const staticKey = "_meta";
```

Comment:
I think this is wrong. The latest version gets the info from `countries` ...

Reply:
Yes, that method is unused. I'll remove it.